resources/route53_domains.go

```diff
@@ -0,0 +1,492 @@
+package resources
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/route53domains"
+	"github.com/aws/aws-sdk-go-v2/service/route53domains/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func Route53Domains() *schema.Table {
+	return &schema.Table{
```

Comment:
Missing ignore error

Reply:
Added:

		IgnoreError:  client.IgnoreAccessDeniedServiceDisabled,

Also added PK on this and relation table